docs/deployment/gunicorn.rst

```diff
@@ -0,0 +1,32 @@
+********
+gunicorn
+********
+
+The short story
+===============
+Running your pyramid based application with gunicorn can be as easy as::
+
+    gunicorn --paste production.ini
+
```

Comment:
IIRC it is indeed possible to add gunicorn settings to the `ini` file too. I will check and add it to the information on the page and let you know :-)